scripts/vagrant/git-dataverse.sh

```diff
@@ -1,31 +0,0 @@
-#!/bin/bash -e
-
-echo "generating ssh keys"
-/usr/bin/mkdir /root/.ssh
-/usr/bin/ssh-keygen -t dsa -N "" -f /root/.ssh/id_dsa
-/usr/bin/ssh-keygen -t dsa -N "" -f /home/vagrant/.ssh/id_dsa
-/usr/bin/chown vagrant /home/vagrant/.ssh/id_dsa
-
-echo "accepting ssh key for 127.0.0.1. don't use localhost because of ssh/ansible."
-/usr/bin/ssh-keyscan -H 127.0.0.1 >> /root/.ssh/known_hosts
-/usr/bin/ssh-keyscan -H 127.0.0.1 >> /home/vagrant/.ssh/known_hosts
-/usr/bin/chown vagrant /home/vagrant/.ssh/known_hosts
-
-echo "copying public key to authorized_keys file"
-/usr/bin/cat /root/.ssh/id_dsa.pub >> /home/vagrant/.ssh/authorized_keys
-/usr/bin/cat /home/vagrant/.ssh/id_dsa.pub >> /home/vagrant/.ssh/authorized_keys
-
-echo "installing git"
-/usr/bin/yum -y install git
-
-echo "punching firewall holes for common dataverse ports"
-/usr/bin/firewall-cmd --permanent --add-port=80/tcp --zone=public
-/usr/bin/firewall-cmd --permanent --add-port=443/tcp --zone=public
-/usr/bin/firewall-cmd --permanent --add-port=8080/tcp --zone=public
-/usr/bin/firewall-cmd --reload
-
-echo "cloning dataverse-ansible repository"
-/usr/bin/git clone https://github.com/IQSS/dataverse-ansible.git /home/vagrant/dataverse-ansible
```

Comment:
Vagrant's synced folders provide a better solution than git-cloning the code into place, as it allows editing and testing the role interactively from the host machine.